include/oneapi/tbb/flow_graph.h

```diff
@@ -3318,6 +3318,11 @@ class overwrite_node : public graph_node, public receiver<T>, public sender<T> {
         spin_mutex::scoped_lock l( my_mutex );
         if (my_buffer_is_valid && is_graph_active( my_graph )) {
             // We have a valid value that must be forwarded immediately.
+            if (dynamic_cast<continue_receiver*>(&s)) {
+                my_successors.register_successor( s );
+                s.try_put( my_buffer );
```

Comment:
I think it makes sense to add a comment that we need to increase predecessors count in continue_node before putting the element. 
I also though about adding an assertion that `try_put` can't fail, but I am not sure it makes a lot of sense.